src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -119,6 +119,10 @@ public FeatureOnOffOptions()
             FeatureName, "AutomaticallyCompleteStatementOnSemicolon", defaultValue: true,
             storageLocation: new RoamingProfileStorageLocation("TextEditor.AutomaticallyCompleteStatementOnSemicolon"));
 
+        public static readonly Option2<bool> AutomaticallyFixStringContentsOnPaste = new(
```

Comment:
`AutomationObject` needs to be updated.

Reply:
Enh.. but does it really?

Reply:
@CyrusNajmabadi First I think I should get to understand how the user will be able to change this option? I see `defaultValue: true` but no VS option to change it? If we add an option in VS then we need it to be persisted when exporting settings.

Reply:
@Youssef1313 haven't done any of the UI options side of this at all.  this is purely the text manipulation side :)

Reply:
done.